spec/controllers/redirections_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe RedirectionsController do
+  describe "GET :next" do
+    it "redirects to the next site" do
+      gabe = create(:redirection, :gabe)
+      edward = create(:redirection, :edward, next: gabe)
+      gabe.update(next: edward)
+
+      get :next, slug: "gabe"
+
+      expect(response).to redirect_to("http://edwardloveall.com")
```

Comment:
Yeah this is more brittle but everytime I write expectations with variables they end up too clever.